crypto.go

```diff
@@ -70,7 +70,7 @@ type PublicSigningKeys Keys
 // returning the signing keys and error (if any).
 func GenerateSigningKeys() (SigningKeys, error) {
 	var signingKeys SigningKeys
-	signingKeyBytes, privateSigningKeyBytes, err := box.GenerateKey(rand.Reader)
+	signingKeyBytes, privateSigningKeyBytes, err := sign.GenerateKey(rand.Reader)
```

Comment:
would any consuming services have problems if they used this method?

Reply:
this was a bug that just got added. This fixes the bug